extensions/stackdriver/stackdriver.h

```diff
@@ -119,6 +119,11 @@ class StackdriverRootContext : public RootContext {
 
   bool shouldLogThisRequest(::Wasm::Common::RequestInfo& request_info);
 
+  // Indicates whether to export server audit log or not.
+  bool enableAuditLog();
+
+  bool shouldAuditThisRequest();
```

Comment:
can you add comment for this function too?

Reply:
Yep.